miqa/core/rest/permissions.py

```diff
@@ -37,7 +37,7 @@ def _wrapped_view(viewset, *args, **wrapped_view_kwargs):
             user_perms_on_project = get_perms(user, project)
             review_perm = has_review_perm(user_perms_on_project)
             read_perm = has_read_perm(user_perms_on_project)
-            error_response = Response(status=status.HTTP_401_UNAUTHORIZED)
+            error_response = Response(status=status.HTTP_403_FORBIDDEN)
```

Comment:
Why construct this in advance? Can we just do it on line 47 when we know we need to return it?

Reply:
Sure, changed in [a31191a](https://github.com/OpenImaging/miqa/pull/478/commits/a31191aef35795936d00ec1e22910cf8c30aa8cd)